.changeset/famous-oranges-go.md

```diff
@@ -0,0 +1,13 @@
+---
+'@swisspost/design-system-documentation': minor
+---
+
+Added label to indicate status of components in foundations, raw components, components, utilities, templates on documentation.
+
+Status label indicates the state of the component:
+- In Progress ⏳: This component has open issues or tasks that need to be fixed before it can be used.
+- Experimental 🆕*: This component has been newly introduced and you might experience some bumps while using it.
+- Stable ✅: This component is tested, has some mileage, other projects are already successfully implementing it, it's safe to use this.
```

Comment:
Input from @sandra-post: stable components should have no icon to reduce mental load.

Reply:
In this case, the icon could be placed after the text to prevent misalignment.